swaglyrics/cli.py

```diff
@@ -42,6 +42,10 @@ def stripper(song: str, artist: str) -> str:
 		else:
 			artist += f'-and-{ar}'
 	song_data = artist + '-' + song
+	if len(artist) < 1:
```

Comment:
If the length of a string is less than 1 then it can only be 0, if it is 0 then you can use `if not artist`